include/swift/AST/Decl.h

```diff
@@ -2854,8 +2854,9 @@ class NominalTypeDecl : public GenericTypeDecl, public IterableDeclContext {
   ///
   /// \param ignoreNewExtensions Whether to avoid loading any new extension.
   /// Used by the module loader to break recursion.
-  ArrayRef<ValueDecl *> lookupDirect(DeclName name,
-                                     bool ignoreNewExtensions = false);
+  SmallVector<ValueDecl *, 4> lookupDirect(
```

Comment:
I think it makes sense to return a TinyPtrVector here. There are a *lot* of cases where there's just one element.